incubator-calls.md

```diff
@@ -0,0 +1,148 @@
+incubation calls are meant to facilitate feedback. Because time itself during
+the meeting is limited, if a particular topic looks of interest to a
+contributor, and they would like to contribute feedback, the facilitators
+encourage contributors to page some context in.
+
+To constructively enable the meeting contributors to have the appropriate
+context, topics and related resources are placed on the agenda, for all
+participants to read ahead of the meeting. Additionally, participants are
+actively encouraged to ask true clarifying questions, more than in a normal TC39
+meeting. For example, "how does this part work?" or "what do you mean by that
+term?". These questions are very useful to champions in refining how they
+present proposals.
+
+### Facilitators
+
+4 days before the call, facilitators will go through the agenda and make a
+judgment call on what to discuss at the next call. Overflow will be pushed to
+the next meeting.
+
+### No FOMO
+
+Participants should feel empowered to skip a call if no agenda items of interest
+to them are present 4 days before the call. Also remember that no binding
+decisions are made in incubator calls. Consensus decisions are made in TC39
+meetings.
+
+Facilitators will conclude the call early instead of using additional time for
+impromptu topics, in case those topics are of interest to participants who
+decided to skip. Instead, impromptu topics will be put on the agenda for future
+calls.
+
+## During the Call
+
+During the call, the agenda items will be processed in order. Unlike plenary,
+there are no hard timeboxes. Instead, the facilitators and champions may ask to
+move on to the next topic at their discretion. Discussions are facilitated by
+TCQ, like in plenary. Notes are taken, like in plenary.
+
+Meeting facilitators are specifically empowered to keep the discussion within
+the scope of the current agenda item. For example, if we're talking about one
+area of concern, and participants bring up another, facilitators may decide to
+queue that new area to be discussed later in the meeting. Or, if participants
+bring up an explicitly out-of-scope item, the facilitators may remind
+participants about the meeting scope.
+
+General timeline:
+1. Quick introduction
+1. Adopting the agenda
+1. Agenda items
+1. Heads up announcements for next call
+
+For each agenda item:
+1. Participant provide ~5 min intro and context-setting
+1. Discussion
+1. Identify action items for champions and core stakeholders, such as following up on a thread
+
+Remember, the guiding principle is to present and receive relevant feedback in a
+more timely manner. When in doubt, yield time to other participants to present
+their feedback instead of attempting to resolve the feedback in the
+moment. Resolving the issue should ideally be done asynchronously, or in plenary
+itself, if particularly controversial.
+
+# After the Call
+
+Participants are encouraged to review the notes, which will be published 7 days
```

Comment:
Where will this be published?

Reply:
I was hoping a new top-level folder in tc39/notes called "incubator-calls".